README.md

```diff
@@ -74,6 +74,19 @@ True
 
 ```
 
+If you don't want to read the whole file, but instead only the first
+row so you have information on the columns:
```

Comment:
"If you don't want to read the whole file, but instead only the first
row so you have information on the columns:"

Can you fix this sentence for clarity?